examples/k8s-bookinfo.yaml

```diff
@@ -56,8 +56,10 @@ spec:
           value: http://$(CONTROLLER_SERVICE_HOST):$(CONTROLLER_SERVICE_PORT)
         - name: A8_DISCOVERY_BACKEND
           value: kubernetes
-        - name: A8_SERVICE
-          value: gateway:none
+        - name: A8_KUBERNETES_POD_NAME
```

Comment:
If the gateway is going to be launched with
A8_SERVICE=gateway:service=gateway
we need to update the JSON files in examples folder related to Gremlin. And the docker examples as well